src/components/Editor/index.js

```diff
@@ -598,24 +615,6 @@ class Editor extends PureComponent {
     return <Preview editor={this.state.editor} />;
   }
 
-  renderInScopeLines() {
```

Comment:
this was a perf hit for no reason. we no longer do anything on the presentation side w/ the `in-scope` lines.